src/main/scala/com/box/castle/router/kafkadispatcher/KafkaDispatcher.scala

```diff
@@ -99,6 +99,10 @@ class KafkaDispatcher(boxSimpleConsumerFactory: CastleSimpleConsumerFactory,
         log.error(s"Broker is unreachable: $broker due to: ${t.getMessage}", t)
         context.parent ! BrokerUnreachable(broker)
       }
+      case UnknownTopicPartition(topicAndPartition) => {
+        log.error(s"Queue processor encountered unknown topic partition for: ${topicAndPartition}")
```

Comment:
log.error(s"KafkaDispatcher encountered unknown topic partition for: ${topicAndPartition}")

Reply:
Done